frontend/fingertips-frontend/app/chart/page.tsx

```diff
@@ -98,10 +107,37 @@ export default async function ChartPage(
       selectedAreasData
     );
 
+    seedData[`availableAreas`] = availableAreas ?? [];
+
     if (updatedSearchState) {
       stateManager.setState(updatedSearchState);
     }
 
+    const compareAreasQueryParams = compareAreasTableRequestParams(
+      searchState,
+      availableAreas ?? []
+    );
+    const compareAreasQueryKey = queryKeyFromRequestParams(
+      EndPoints.HealthDataForAnIndicator,
+      compareAreasQueryParams
+    );
+
+    if (
+      compareAreasTableIsRequired(searchState) &&
+      !Object.keys(seedData).includes(compareAreasQueryKey)
```

Comment:
Can this check also be included as part of `compareAreasTableIsRequired` function?